rdsa_utils/cdp/helpers/s3_utils.py

```diff
@@ -1098,3 +1100,70 @@ def load_json(
         raise Exception(error_message) from e
 
     return data
+
+
+def write_csv(
+    client: boto3.client,
+    bucket_name: str,
+    data: pd.DataFrame,
+    filepath: str,
+    **kwargs,
+) -> bool:
+    """
+    Write a Pandas Dataframe to csv in an s3 bucket.
+
+    Uses StringIO library as a RAM buffer, so at first Pandas writes data to the
+    buffer, then the buffer returns to the beginning, and then it is sent to
+    the s3 bucket using the boto3.put_object method.
+
+    Parameters
+    ----------
+    client : boto3.client
+        The boto3 S3 client instance.
+    bucket_name : str
+        The name of the S3 bucket.
+    data : pd.DataFrame
+        The dataframe to write to the spexified path.
+    filepath : str
+        The filepath to save the dataframe to.
+    kwargs : dict
+        Optional dictionary of Pandas to_csv arguments.
```

Comment:
@zorge69 : could you please remove the type-hints from the docstring.

Reply:
I have removed the type-hints